include/picongpu/particles/Particles.tpp

```diff
@@ -79,44 +80,50 @@ Particles<
     ),
     m_datasetID( datasetID )
 {
-    size_t sizeOfExchanges = 2 * 2 * ( BYTES_EXCHANGE_X + BYTES_EXCHANGE_Y + BYTES_EXCHANGE_Z ) + BYTES_EXCHANGE_X * 2 * 8;
+    using ExchangeMemCfg = GetExchangeMemCfg_t< Particles >;
+
+    size_t sizeOfExchanges = 2 * 2 * (
```

Comment:
good time to:
- declare or document what `2*2` and `2*8` stand for
- make `sizeOfExchanges` a `const`

Reply:
I have changed the way how the size of all exchanges is calculated, this should be much better understandable than a comment over the ugly magic numbers.

Reply:
btw: Maybe the calculated amount was wrong for 3D or 2D because the old equation was equal for 2D and 3D